api/src/main/java/org/openmrs/api/db/ConditionDAO.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db;
+
+import java.util.List;
+
+import org.openmrs.Condition;
+import org.openmrs.Patient;
+
+/**
+ * This interface defines database methods for condition objects. 
+ *
+ * <ul>
+ *     <li>{@link #getActiveConditions(Patient)}</li>
```

Comment:
Do not list them here because it will create a maintenance problem of having to keep them is sync with new ones, updates, and deletes.

Reply:
@dkayiwa, alright. Let me change that right now.